option_test.go

```diff
@@ -830,8 +830,117 @@ func TestOptionDescription(t *testing.T) {
 			option.Middleware(dummyMiddleware), // After the description (6th middleware)
 			option.Middleware(dummyMiddleware), // 7th middleware, should not be included
 		)
+		require.Equal(t, "#### Controller: \n\n`github.com/go-fuego/fuego_test.helloWorld`\n\n#### Middlewares:\n\n- `github.com/go-fuego/fuego.defaultLogger.middleware`\n- `github.com/go-fuego/fuego_test.dummyMiddleware`\n- `github.com/go-fuego/fuego_test.dummyMiddleware`\n- `github.com/go-fuego/fuego_test.dummyMiddleware`\n- `github.com/go-fuego/fuego_test.dummyMiddleware`\n\n---\n\nanother description", route.Operation.Description)
+	})
+
+	t.Run("Disable middleware section", func(t *testing.T) {
+		s := fuego.NewServer(
+			fuego.WithEngineOptions(
+				fuego.WithMiddlewareConfig(
+					fuego.MiddlewareConfig{
+						IsInititalized:           true,
+						DisableMiddlewareSection: true,
+						ShortMiddlewaresPaths:    false,
+						MaxNumberOfMiddlewares:   6,
+					},
+				),
+			),
+		)
```

Comment:
Can you add a text with `fuego.MiddlewareConfig{ }` without specifying the `MaxNumberOfMiddlewares` please?